Maker-Agents-Suit-Module/OSMAgent/src/big.queued.price.deviation.ts

```diff
@@ -1,31 +1,21 @@
-import {
-  Log,
-  TransactionEvent,
-  Trace,
-  HandleTransaction,
-  Finding,
-  FindingType,
-  FindingSeverity,
-} from "forta-agent";
-import Web3 from "web3";
+import { BigNumber } from "ethers";
+import { Interface, parseBytes32String } from "ethers/lib/utils";
+import { TransactionEvent, Trace, HandleTransaction, Finding, FindingType, FindingSeverity } from "forta-agent";
+
 import AddressesFetcher from "./addresses.fetcher";
 
 const PEEK_FUNCTION_SELECTOR = "0x59e02dd7";
-const LOG_VALUE_EVENT_SIGNATURE = "LogValue(bytes32)";
+const LOG_VALUE_EVENT_SIGNATURE = "event LogValue(bytes32)";
```

Comment:
`FilterFunction` and `FilterLog` functions from forta-agent require the `function` and `event` keywords in their first argument (fuctionAbi/ eventAbi). They return an error otherwise